server/todo-api/schemas/to-do-schema.ts

```diff
@@ -0,0 +1,15 @@
+import { model, Schema } from 'mongoose';
+import { ToDo } from '../model/todo-model';
+
+const ToDoSchema = new Schema<ToDo>({
+  createdBy: { type: Schema.ObjectId },
+  isCompleted: { type: Boolean, default: false },
+  description: { type: String },
+  priority: { type: Number, max: 10, min: 0 },
+  title: { type: String, required: true },
+  dueDate: { type: Date },
+});
+
+const ToDoModel = model('ToDo', ToDoSchema);
```

Comment:
Esto se convierte en el nombre de la colección de mongo y debemos evitar los problemas de mayusculas minúsculas
```suggestion
const ToDoModel = model('todo', ToDoSchema);
```